storage/pebble/lookup.go

```diff
@@ -14,11 +14,34 @@ type lookupKey struct {
 	encoded []byte
 }
 
+func createHeightKey(identifier byte) []byte {
```

Comment:
we might need to add data type prefix to be able to expand

Reply:
my opinion here as that we should not, since `SeekPrefixGE` will have to have a secondary filter for the data type prefix and then apply the `key/owner` filter, making the lookup 2 scans instead of 1. My solution is to use a second pebble instance with a second directory for general purpose storage.